restore.go

```diff
@@ -165,7 +165,10 @@ func restore(filename string) error {
 }
 
 func createContainer(backup Backup) (string, error) {
-	fmt.Println("Restoring Container with hostname:", backup.Config.Hostname)
+	var nameparts = strings.Split(backup.Name, "/")
```

Comment:
If you let the compiler infer the type anyway, I think one should prefer this form:

```golang
nameparts := strings.Split(backup.Name, "/")
```

Same for the line below :smiley:

Reply:
Thanks, yep I really haven't worked with golang before so fumbled my way though basic things like this!